src/DacpacTool/Extensions.cs

```diff
@@ -141,8 +141,27 @@ public static void AddReference(this TSqlModel model, string referencePath, stri
 
             var crossPlatformPath = referencePath.Replace('\\', '/');
 
-            var customData = Activator.CreateInstance(Type.GetType("Microsoft.Data.Tools.Schema.SchemaModel.CustomSchemaData, Microsoft.Data.Tools.Schema.Sql"), "Reference", "SqlSchema");
+            var customType = Type.GetType("Microsoft.Data.Tools.Schema.SchemaModel.CustomSchemaData, Microsoft.Data.Tools.Schema.Sql");
```

Comment:
Looking at this again now I'm wondering if we should do this statically, instead of doing this every time we add a reference. We could add a static constructor that resolves these types and methods.

Reply:
Not 100% sure, but attempted an update for this